docs/concepts/whatisvpp/ndrrates.rst

```diff
@@ -0,0 +1,53 @@
+.. _ndrrates:
+
+.. toctree::
+    
+NDR Rates 
+***********************************
+
+NDR rates for 2p10GE, 1 core, L2 NIC-to_NIC
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following chart shows the NDR rates on: 2p10GE, 1 core, L2
+NIC-to_NIC.
+
+.. figure:: /_images/Vpp_performance_barchart_ndr_rates_l2-nic-to-nic.jpg
```

Comment:
Right these performance graphs are older than me. 
We need to incorporate latest performance numbers from Cisco.

Include the graph  https://docs.fd.io/csit/rls1804/report/_static/vpp/64B-1t1c-l2-sel1-ndrdisc.html. I mean include, don't link.

Reply:
Cisco = CSIT